homeassistant/components/asuswrt/router.py

```diff
@@ -35,8 +106,16 @@ def __init__(self, hass: HomeAssistantType, entry: ConfigEntry) -> None:
         """Initialize a AsusWrt router."""
         self.hass = hass
         self._entry = entry
+        self._unique_id = entry.unique_id or DOMAIN
```

Comment:
There's no config entry unique_id set for this integration.

Reply:
I write this to be able to manage unique_id in case in the future an unique_id (as mac address) for the router can be provided. For the moment, because is empty, I use DOMAIN that is unique because multiple instance are not allowed. Is this a problem?

Reply:
Let's not add it until there's an actual unique id.